### PR TITLE
MINOR: Updates from running spotless apply

### PR DESCRIPTION
**Pull request by bbejeck**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: streams/src/test/resources/log4j.properties [hunk 1]
```diff
@@ -26,6 +26,7 @@ log4j.logger.org.apache.kafka.clients=ERROR
 # These are the only logs we will likely ever find anything useful in to debug Streams test failures
 log4j.logger.org.apache.kafka.clients.consumer=INFO
 log4j.logger.org.apache.kafka.clients.producer=INFO
+log4j.logger.org.apache.kafka.common.telemetry.internals.ClientTelemetryUtils=DEBUG
```
Comment (lucasbru) [inline streams/src/test/resources/log4j.properties]: That doesn't belong in this PR, does it?

**Review by lucasbru — COMMENTED** [review]
one change looks unrelated. Otherwise LGMT

File: streams/src/test/resources/log4j.properties (hunk 1 above)
Comment (bbejeck) [reply streams/src/test/resources/log4j.properties]: Yeah - I don't know where that came from

**Review by cadonna — APPROVED** [review]
LGTM!
